src/Pages/sunkyungnoh/Login/components/Footer.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './Footer.scss';
+
+class Footer extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      footer: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/sunkyungnoh/footerData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(footerData => {
+        this.setState({
+          footer: footerData,
+        });
+      });
```

Comment:
```suggestion
    fetch('/data/sunkyungnoh/footerData.json')
      .then(res => res.json())
      .then(footer => this.setState({ footer }));
```
- get method 는 기본값이기 때문에 생략해줄 수 있습니다
- 변수명과 객체의 키 값이 같을 경우 객체 단축 속성명을 사용하면 보다 간단히 객체를 표현할 수 있습니다

Reply:
😮 와우 전부 수정했습니다!